runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/users/register/index.ts

```diff
@@ -0,0 +1,9 @@
+import tableRegisterTranslations from './table';
+import viewRegisterTranslations from './view';
+
+const usersViewTranslations = {
+    ...viewRegisterTranslations,
+    ...tableRegisterTranslations,
```

Comment:
this register is stabbing my soul.
I suggest to take a while to rename all registry files and function names to registration - now you do have far fewer changes and up-to-date branch

Reply:
F2

Reply:
OK.